modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+
+export * as CommerceComponents from './components/index';
+export { default as CommerceServiceProvider } from './ServiceProvider/index';
+export { default as CommerceFrontendUtils } from './utilities/interface/index';
```

Comment:
@marco-leo @FabioDiegoMastrorilli @garofalog

Here's the interface for the whole module. As said above, `CommerceComponents` includes the `MiniCartContext` exposed from our list of components and is the only one that I need for SaaS for the moment being.